autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/CronTimeEvaluator.java

```diff
@@ -67,11 +78,15 @@ private boolean isPolicyAttached(BaseAlert alert) {
 
     @Override
     public void run() {
-        long start = System.currentTimeMillis();
-        Cluster cluster = clusterService.find(clusterId);
-        MDCBuilder.buildMdcContext(cluster);
-        publishIfNeeded(alertRepository.findAllByCluster(clusterId));
-        LOGGER.info("Finished cronTimeEvaluator for cluster {} in {} ms", clusterId, System.currentTimeMillis() - start);
+        try {
+            long start = System.currentTimeMillis();
```

Comment:
Can move out of try, like AmbariAgentHealthEvaluator

Reply:
I moved it out of the try block